src/dashboard/components/options/OptionsHome.js

```diff
@@ -35,24 +37,24 @@ export default function OptionsHome(props) {
     }
   }, [preloading, loading])
 
-  // const onChangeBackupLocation = (event) => {
-  //   let file = event.target.files[0]
-  //   if (file) {
-  //     let folder = file.path.split('/')
-  //     let folderPath = folder.slice(0, folder.length - 1).join('/')
-  //     setBackupLocation(folderPath)
-  //     saveSetting('user.backupLocation', folderPath)
-  //   }
-  //   setLoading(false)
-  //   setPreloading(false)
-  // }
+  const onChangeBackupLocation = () => {
+    const title = t('Choose your backup location')
+    const properties = ['openDirectory']
+    const files = dialog.showOpenDialogSync(win, { title, properties })
+    if (files && files.length) {
+      let folderPath = files[0]
+      setBackupLocation(folderPath)
+      saveSetting('user.backupLocation', folderPath)
+    }
+    setLoading(false)
```

Comment:
Why do we unset loading and pre-loading here?

Reply:
I spoke to @dstonem about `loading` and `preloading`, and he said that it was part of another attempt at this feature.
I'd recommend removing lines 21-38 (from where `loading` is defined up until the `useEffect`, which uses them) and then removing the `blur` and `focus` handlers in the `div` at the root of this component's render tree.